pkg/generate/git/repository.go

```diff
@@ -465,3 +465,19 @@ func IsExitCode(err error, exitCode int) bool {
 	}
 	return false
 }
+
+func gitBinary() string {
+	if runtime.GOOS == "windows" {
+		return "git.exe"
+	}
+	return "git"
+}
+
+func IsGitInstalled() bool {
```

Comment:
That it's used from the timedCommand function above where you could have specified a git binary to use.